java/src/main/java/com/databricks/mlflow/sagemaker/JavaFunc.java

```diff
@@ -0,0 +1,74 @@
+package com.databricks.mlflow.sagemaker;
```

Comment:
Included this file for reference about how `java_function` deserialization may be implemented. It is package-private to the `sagemaker` package. Please let me know if you want this removed.